h/static/styles/partials-v2/_group-invite.scss

```diff
@@ -9,22 +9,16 @@
 }
 
 .group-invite__input {
-  padding: 5px;
-  padding-top: 0;
   padding-bottom: 0;
+  padding-left: 5px;
+  padding-right: 22px;  /* Make space for the clipboard icon. */
+  padding-top: 0;
   margin-top: 12px;
   margin-bottom: 8px;
   border: 1px solid $grey-3;
   border-radius: 2px;
   width: 100%;
   height: 31px;
-
-  .env-js-capable & {
```

Comment:
For reference, the plain HTML version of this component now looks like this:

![group-invite-plain-html](https://cloud.githubusercontent.com/assets/2458/20346532/73a1c9d4-abf4-11e6-92e2-c5275026bb7d.png)